### PR TITLE
Pr add vgsx

### PR DESCRIPTION
**Pull request by al1img**

### CODE REVIEW

File: tools/libxl/libxl_create.c [hunk 1]
```diff
@@ -1452,6 +1452,7 @@ const struct libxl_device_type *device_type_tbl[] = {
     &libxl__dtdev_devtype,
     &libxl__vdispl_devtype,
     &libxl__vsnd_devtype,
+	&libxl__vgsx_devtype,
```
Comment (andr2000) [inline tools/libxl/libxl_create.c]: white space

File: tools/libxl/libxl_vgsx.c [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2016 EPAM Systems Inc.
```
Comment (andr2000) [inline tools/libxl/libxl_vgsx.c]: 2017

**Comment by andr2000** [comment]
lgtm, rb

**Comment by iartemenko** [comment]
@al1img pls update rb

**Comment by andr2000** [comment]
@iartemenko pls merge

**Pull request merged**